submission/loglist_refresher_test.go

```diff
@@ -90,6 +92,22 @@ func TestNewLogListRefresherNoFile(t *testing.T) {
 	}
 }
 
+func TestNewCustomLogListRefresher(t *testing.T) {
+	f, err := createTempFile(`{"operators": [{"id":0,"name":"Google"}]}`)
+	if err != nil {
+		panic(err)
+	}
+	defer os.Remove(f)
+
+	client := &http.Client{Timeout: time.Second}
+
+	llr := NewCustomLogListRefresher(client, f)
```

Comment:
I was thinking something like below, which would give you a testable side-effect to the implementation of LLR actually using the client (your test does indeed verify that the c'tor stored it correctly, but it doesn't check that it's used by `Refresh()`):

```golang
type fakeTransport struct {
  called bool
}

func (ft *fakeTransport) RoundTrip(req *http.Request) (*http.Response, error) {
  ft.called = true 
  return nil, fmt.Errorf("Fake News")
}

func TestPassedInClientIsUsedForRefresh() {
	fakeTransport := fakeTransport{}
       c := http.Client{
		Transport: &fakeTransport,
		}
   
      llr := NewCustom...(c, ...)
      llr.Refresh() // or whatever
      
	if !fakeTransport.called  {
	   t.Fatalf("Oh noes!")
	} 
}
```

Or could use a Mock for the `Transport` if you wanted to double check stuff like the path.

Reply:
Thanks, added the test according to the comment.

Reply:
neat - thanks!